src/sentry/api/endpoints/group_events.py

```diff
@@ -91,6 +91,10 @@ def respond(queryset):
 
             events = events.filter(q)
 
+        # TODO currently snuba can be used to get this filter of event_ids matching
+        # the search tags, which is then used to further filter a postgres QuerySet
+        # Ideally we would just use snuba to completely replace the fetching of the
+        # events.
```

Comment:
This comment is originally from #10025, I was originally planning on moving event search into the search backend but that seemed like overkill at this point and there were API decisions that I didn't want/need to make at this point.